pulse/urls/__init__.py

```diff
@@ -16,6 +16,6 @@
     path('projects/', include(project_urls)),
     path('questions/', include(question_urls)),
     path('users/', include(user_urls)),
-    path('tags/', include (tag_urls))
+    path('tags/', include (tag_urls)),
     path('comments/', include(comment_urls))
```

Comment:
for future reference, I think python lets you put a `,` at the end of a list. Not sure if this is best practice or not, but could be useful for this type of error!